src/adapter/src/catalog/storage.rs

```diff
@@ -1036,6 +1036,14 @@ impl<'a, S: Append> Transaction<'a, S> {
         Ok(())
     }
 
+    pub fn remove_timestamp(&mut self, timeline: Timeline) {
+        let n = self
```

Comment:
Maybe `let timeline = timeline.to_string();` instead of calling it in a loop?

Reply:
Fixed